### PR TITLE
created product page

### PR DESCRIPTION
**Pull request by sagargse**
Listing of all products
image
image

### CODE REVIEW

File: cssfiles/cards.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
+.vertical-card{
+    background-color: var(--primary-color);
+    color: var(--primary-text-color);
+    box-shadow: 0 4px 8px 0 var(--primary-color);
```
Comment (NeerajVodala) [inline cssfiles/cards.css:4]: using variable for box shadow is recommended(so that you can use it elsewhere)

**Review by NeerajVodala — APPROVED** [review]
Use a variable for box shadow(if you think it is necessary and helpful) before merging.
All the best.

File: cssfiles/cards.css (hunk 1 above)
Comment (sagargse) [reply cssfiles/cards.css:4]: ok will consider that

**Review by Mahi24-prog — APPROVED** [review]
You can write flex related classes once and use whenever you need.

**Comment by sagargse** [comment]
> You can write flex related classes once and use whenever you need.

ok i will try that

**Pull request merged**